sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -139,6 +139,28 @@ DOCS=( CONTRIBUTING.md README.md )
 
 CHECKREQS_DISK_BUILD="3G"
 
+MY_CUDA_ARCH=
+
+get_cuda_arch() {
+	if [[ -z ${CUDA_ARCH} ]]; then
+		MY_CUDA_ARCH=native
+		return
+	else
```

Comment:
This if/else feels kind of redundant, you're setting =native in two spots.

Default variable assignment may be of use, e.g. `${CUDA_ARCH:-all}` would return `all` if unset, otherwise `${CUDA_ARCH}`

Reply:
I was thinking on setting the MY_CUDA_ARCH to an initial value of `native` to avoid this path.
Good point about the default assignment

Reply:
On a closer look... together with `test -z` the test would never be true, because, if `CUDA_ARCH` is unset or null, the default value will be assigned, thus giving the string a length.